crates/indexer/src/util.rs

```diff
@@ -168,11 +168,9 @@ mod tests {
 
     #[test]
     fn test_parse_timestamp() {
-        let current_year = chrono::offset::Utc::now().year();
-
         let ts = parse_timestamp(1649560602763949, 1);
         assert_eq!(ts.timestamp(), 1649560602);
-        assert_eq!(ts.year(), current_year);
+        assert_eq!(ts.year(), 2022);
```

Comment:
Yes, I cherry-picked [this](https://github.com/aptos-labs/aptos-core/pull/6156/commits/ce68cffc5a0d703dd8b5d4fdb3a70c4c300f95c9) fix because the test failed

Reply:
Gotcha, thanks @rahxephon89! 😄